src/osd/OSD.cc

```diff
@@ -759,8 +759,8 @@ void OSDService::update_osd_stat(vector<int>& hb_peers)
 {
   Mutex::Locker lock(stat_lock);
 
-  osd_stat.hb_in.swap(hb_peers);
-  osd_stat.hb_out.clear();
+  osd_stat.hb_out.swap(hb_peers);
```

Comment:
can we take this chance to remove `hb_in` in a separated commit?

@liewegas what do you think?